publons/PublonsPlugin.inc.php

```diff
@@ -22,9 +22,9 @@ class PublonsPlugin extends GenericPlugin {
      * @return boolean True iff plugin initialized successfully; if false,
      *  the plugin will not be registered.
      */
-    function register($category, $path) {
+    function register($category, $path, $mainContextId = null) {
```

Comment:
this change is because the parent class input parameters is changed, so should keep consistant